### PR TITLE
PLU-241: [name-your-pipe-3]: remove connections and routes

### PR DESCRIPTION
**Pull request by m0nggh**
## Problem
We have too many "name your pipe" pipes and it is hard for us to tell what the user is doing at one glance, so we should make the user name their pipe so that it is easier for them to track their own pipes as well.

## Details
This PR focuses on removing the connections functionalities and routes on the `apps` page
Note: didn't migrate all the MUI components because the `apps` page will go through a revamp in the near future

Frontend:
- Removed add connections in apps page
- Deleted the AddNewAppConnection component
- Removed `searchinput` for `Applications`
- Remove add connection in each connection
- Updated the `NoResultFound` component with new graphic and removed MUI

Backend:
- Removed `appName` from the `get-connected-apps` graphQL query

## Tests
- [x] Sanity check that pipe creation and execution still works
- [x] NoResultFound appears for where it is supposed to be
- [x] Apps page cannot allow for adding of connections and flows anymore

### CODE REVIEW

**Comment by m0nggh** [comment]
* **#683** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/683?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#682** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/682?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#681** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/681?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `develop-v2`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @m0nggh and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by ogp-weeloong — APPROVED** [review]
LGTM pls update with copy suggestions from stacey

**Review by ogp-weeloong — COMMENTED** [review]
Can you do a quick check on how many hits we have on the removed routes over the past 1 month? We should let jackson know of this upcoming change, and also the blast radius / expected num of people who might complain

**Review by m0nggh — COMMENTED** [review]
ok i made the copy changes and checked it with Stacey alr, approved!

and regarding the `/editor/create` route: i see about 258 being made the last week but it was because of clicking on the `create pipe` button so it is normal... only 1 user went immediately to that page so i think should not be a concern

File: packages/frontend/src/pages/Application/index.tsx [hunk 1]
```diff
@@ -180,13 +142,6 @@ export default function Application(): React.ReactElement | null {
       </Box>
 
       <Routes>
-        <Route
-          path="/connections/add"
-          element={
-            <AddAppConnection onClose={goToApplicationPage} application={app} />
-          }
-        />
-
```
Comment (pregnantboy) [inline packages/frontend/src/pages/Application/index.tsx:189]: here

**Pull request merged**